integrations/issue_test.go

```diff
@@ -0,0 +1,23 @@
+// Copyright 2017 The Gitea Authors. All rights reserved.
+// Use of this source code is governed by a MIT-style
+// license that can be found in the LICENSE file.
+
+package integrations
+
+import (
+	"net/http"
+	"testing"
+
+	"code.gitea.io/gitea/models"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestNoLoginViewIssue(t *testing.T) {
+	assert.NoError(t, models.LoadFixtures())
+
+	req, err := http.NewRequest("GET", "/user1/repo1/issues/1", nil)
```

Comment:
repo1 is owned by user2 (https://github.com/go-gitea/gitea/blob/master/models/fixtures/repository.yml) -> so /user2/repo1/issues/

Ref : 
- https://github.com/go-gitea/gitea/blob/master/models/fixtures/user.yml
- https://github.com/go-gitea/gitea/blob/master/models/fixtures/issue.yml